SingularityUI/app/templates/vex/taskKillDestroy.hbs

```diff
@@ -0,0 +1,5 @@
+<p>Are you sure you want to destroy this task?</p>
+<pre>{{ id }}</pre>
+<p>
+    This will override cleanup type, immediately initiate a non-graceful kill (-9 PID) and not wait for replacement tasks.
```

Comment:
might want to reword to `This will override any pending cleanup operations and immediately initiate process termination (i.e. kill -9) without waiting for replacement tasks.`